examples/airflow/training_workflow/README.md

```diff
@@ -0,0 +1,164 @@
+# Running SkyPilot tasks in Airflow
+
+
+In this guide, we show how a training workflow involving data preprocessing, training and evaluation can be first easily developed with SkyPilot, and then orchestrated in Airflow.
```

Comment:
I am thinking if we can also link to our docs for skypilot dag, so anyone who have not determined to use airflow, may directly try our native solution: https://skypilot.readthedocs.io/en/latest/examples/managed-jobs.html#job-pipelines

Reply:
Thanks! added a tip.